### PR TITLE
Warn windows users about slow first compile

### PR DESCRIPTION
**Pull request by lostdusty**
Relates to https://canary.discord.com/channels/953020122690359327/953030382993952818/1213112901057912862

### CODE REVIEW

**Review by andydotxyz — CHANGES_REQUESTED** [review]
This "will be faster." sounds a little bit pessimistic.
Would "will be fast." be OK instead?

**Comment by lostdusty** [comment]
Sure

File: README.md [hunk 1]
```diff
@@ -97,6 +97,10 @@ And you can run that simply as:
 
     go run main.go
 
+> [!NOTE]  
+> The first compilation of Fyne on Windows _can_ take up to 10 minutes, depending on your hardware. Subsequent builds will be fast.
+This is caused because the C compiler on Windows lacks optimizations present on Linux and MacOS.
```
Comment (dweymouth) [inline README.md:102]: I'm not sure we want to say what the cause is. We're not 100% certain that it's due to the C compiler (it seems Windows Defender may also play a role?) and even so we might not want to "bad mouth" the Windows C compiler ;)

Comment (lostdusty) [reply README.md:102]: Defender doesn't really do much. I have it uninstalled and it is still slow on the first compile.

**Comment by lostdusty** [comment]
Just noted I accidentally closed this PR; I was trying to organize the translation PR and decided to delete the fork of fyne I already had, but i forgot it was linked to this PR 😬

**Comment by lostdusty** [comment]
Speaking of, I believe using _faster_ instead of _fast_ would be better in my opinion.

**Comment by lostdusty** [comment]
Gotcha.

**Comment by andydotxyz** [comment]
> Just noted I accidentally closed this PR; I was trying to organize the translation PR and decided to delete the fork of fyne I already had, but i forgot it was linked to this PR 😬

Oh I'm sorry, I should probably have asked if it was a mistake before applying the change.

**Pull request closed without merging**